src/chat/Chat.js

```diff
@@ -16,6 +16,12 @@ import UnreadNotice from './UnreadNotice';
 
 
 export default class Chat extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.scrollOffset = 0;
```

Comment:
You can take this out of here and just do a 
```js
scrollOffset = 0;
```
outside the constructor, and remove the constructor itself.